app/code/Magento/Sales/view/adminhtml/templates/order/create/coupons/form.phtml

```diff
@@ -17,8 +17,10 @@
 <div class="admin__field field-apply-coupon-code">
     <label class="admin__field-label"><span><?= /* @escapeNotVerified */ __('Apply Coupon Code') ?></span></label>
     <div class="admin__field-control">
+        <?php if (!$block->getCouponCode()): ?>
         <input type="text" class="admin__control-text" id="coupons:code" value="" name="coupon_code" />
         <?= $block->getButtonHtml(__('Apply'), 'order.applyCoupon($F(\'coupons:code\'))') ?>
+        <?php endif; ?>
```

Comment:
Could be simplified with
`if() {
} else {
}`

Reply:
Also may be make sense to do something like coupon form in cart: disabled input + button
![rqlno_2019-06-19_21-33-01](https://user-images.githubusercontent.com/1129623/59791133-171fda00-92da-11e9-9125-1901faa4a214.jpg)